frontend/lib/laletterbuilder/components/create-account.tsx

```diff
@@ -33,7 +33,6 @@ export const LaLetterBuilderCreateAccount = LaLetterBuilderOnboardingStep(
           initialState={{
             ...BlankLaLetterBuilderCreateAccountInput,
             canWeSms: true,
-            email: "",
```

Comment:
Is there a reason this is being removed? Is it just that it's not needed, but not necessarily connected to the theme of this PR? that's totally ok just wanna make sure I'm not missing something

Reply:
Oops, yeah this is left over from when I was playing around with fixes. Removing